frontend/packages/console-app/src/components/quick-starts/data/install-associate-pipeline-quickstart.ts

```diff
@@ -8,69 +8,63 @@ export const installAssociatePipelineQuickStart = {
   },
   spec: {
     version: 4.7,
-    displayName: `Import an application and associate a pipeline with it`,
+    displayName: `Deploying an application with a pipeline`,
     duration: 10,
     iconURL: pipelineIcon,
-    description:
-      'Install an application, associate a pipeline, start the pipeline and explore the pipeline run',
-    prerequisites: 'OpenShift® Pipelines Operator must be installed',
-    introduction: `### In this tour you will complete 3 tasks:
-1. Create an application from Git.
-2. Explore your application
-3. Explore your Pipeline Run`,
+    description: `Import an application from git, add a pipeline to it, and run the Pipeline.`,
```

Comment:
"Git" is capitalized in the OL after this. Should it be capitalized here, too?

Reply:
🤷 It's capitalized below since we are referencing something in the UI that is capitalized.  Leaving as is, unless someone tells me otherwise :)

Reply:
AFAIK, it's a proper noun and should always be capitalized aside from in CLI commands. 

That's how they write it in https://git-scm.com/about and the output of `git help git`, too. :man_shrugging:

Reply:
+1, Git is always Title case.

Reply:
Opened a bug related to this. 

https://bugzilla.redhat.com/show_bug.cgi?id=1874241

Reply:
Updated